src/index.html

```diff
@@ -9,6 +9,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <label class="logo">
+        <a href="#home" class="pic"></a>
+      </label>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
```

Comment:
```suggestion
            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link is-active">Apple</a>
          </li>

          <li class="nav__item">
            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
              Samsung
            </a>
          </li>

          <li class="nav__item">
            <a href="https://mate-academy-form-lesson.herokuapp.com/create-application" class="nav__link">
              Smartphones
            </a>
          </li>

          <li class="nav__item">
```
pay attention to the code style